packages/core/upload/admin/src/components/EditAssetDialog/PreviewBox/AssetPreview.js

```diff
@@ -15,7 +15,7 @@ const CardAsset = styled(Flex)`
 `;
 
 export const AssetPreview = forwardRef(({ mime, url, name, ...props }, ref) => {
-  const [lang] = usePersistentState('strapi-admin-language', 'en');
+  const lang = 'en';
```

Comment:
I actually think we can just get rid of this and the prop `srcLang` it's not really needed and iirc you need to know the language of the video, which might not be `en`.